azure-pipelines.vrt-pr.yml

```diff
@@ -28,12 +28,32 @@ jobs:
 
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28290&commitId=722f47486aa9f7bc8392929682b05d7c53e8cbd7&env=prod&clientType=FLUENTUI)

_Diffs exceeded the limit allowed for comments. Please use the report link above to check your diffs._


<div id="vrtCommentE2E Tests-v9"/>